index.html

```diff
@@ -4,22 +4,88 @@
 <head>
     <meta charset="UTF-8">
 
-    <title></title>
+    <title>Apple</title>
 
     <link rel="stylesheet" href="css/cssreset.css">
     <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css"
+    integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lato:100,300&display=swap" rel="stylesheet">
 </head>
 
 <body>
     <header>
+        <section class = "main-header">
+            <div>
+                <nav class = "nav-header">
+                    <ul><!--
+                    --><li class = "nav-button nav-start"><a class = "anch" href="#"><i class="fab fa-apple"></i></a></li><!--
+                    --><li class = "nav-button"><a class = "anch" href="#">Store</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">Mac</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">iPod</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">iPhone</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">iPad</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">iTunes</a></li><!--
+                    --><li class = "nav-button"><a class = "anch"  href="#">Support</a></li><!--
```

Comment:
Consider changing the navbar links to buttons.

Reply:
Thanks Bola. I considered the use of buttons in the navbar elements, but then I [changed the buttons to links](https://github.com/mapra99/appleClone/pull/1/commits/87cd5243fe23d91541a5783dca9aa8639f0b7844), following the suggestions made in [this](https://www.w3schools.com/howto/howto_js_topnav.asp) and [this](https://wearefieldtrip.com/journal/css-horizontal-navigation-bar/) article. Which alternative is preferable? Buttons or Links?

Reply:
Those links are instructions on how to create a navbar. Using anchor element (`<a>`) is much easier and preferable in this context. Buttons are mostly used in forms or for call-to-action components. According to Microverse guidelines, you are expected to use buttons but it's okay to use anchor element (`<a>`).